lib/gruf/client/error.rb

```diff
@@ -50,7 +50,7 @@ class InvalidArgument < Validation; end
       class NotFound < Validation; end
       class AlreadyExists < Validation; end
       class OutOfRange < Validation; end
-
+      class ResourceExhausted  < Validation; end
```

Comment:
`ResourceValidation` isn't really a validation-type error - it's meant to be when the server runs out of workers in its thread pool. Can you have this extend `Error` instead? Also, you have an extra space before the `<`.

Reply:
Make sense, thank you, making the changes.